backend/OR-Tools/OR-Tools/Solvers/OrToolsConverter.cs

```diff
@@ -0,0 +1,245 @@
+
+                while (routing.IsEnd(routeIndex) == false)
+                {
+                    long previousRouteIndex = routeIndex;
+
+                    load += solution.Value(capacityDimension.CumulVar(routeIndex));
+
+                    long distance = routing.GetArcCostForVehicle(previousRouteIndex, routeIndex, 0);
+                    IntVar timeVar = timeDimension.CumulVar(routeIndex);
+
+                    itineraries.Add(new Itineraries
+                    {
+                        VehicleName = vehiclesNames[vehicleIndex],
+                        Load = (int)load,
+                        Distance = (int)distance,
+                        From = minDate.AddMinutes(solution.Min(timeVar)),
+                        To = minDate.AddMinutes(solution.Max(timeVar))
+                    });
+
+                    routeIndex = solution.Value(routing.NextVar(routeIndex));
+                }
+            }
+
+            return itineraries;
+        }
+
+        private IList<Summary> GetSummaries(FileInput input, RoutingIndexManager manager, RoutingModel routing, Assignment solution)
+        {
+            int vehiclesAmount = input.Vehicles.Count();
+            IDictionary<int, string> vehiclesNames = input.Vehicles.ToDictionary(k => k.Id - 1, v => v.Name);
+
+            List<Summary> summaries = new List<Summary>();
+
+            RoutingDimension capacityDimension = routing.GetDimensionOrDie("Capacity");
+            RoutingDimension timeDimension = routing.GetMutableDimension("Time");
+
+            for (int vehicleIndex = 0; vehicleIndex < vehiclesAmount; ++vehicleIndex)
+            {
+                long load = 0;
+                long routeDistance = 0;
+                long routeIndex = routing.Start(vehicleIndex);
+                int numberOfVisits = 0;
+
+                while (routing.IsEnd(routeIndex) == false)
+                {
+                    long previousVehicleIndex = routeIndex;
+
+                    load += solution.Value(capacityDimension.CumulVar(routeIndex));
+                    routeDistance += routing.GetArcCostForVehicle(previousVehicleIndex, routeIndex, 0);
+                    ++numberOfVisits;
+
+                    routeIndex = solution.Value(routing.NextVar(routeIndex));
+                }
+
+                load += solution.Value(capacityDimension.CumulVar(routeIndex));
+
+                long vehicleSpentTime = solution.Min(timeDimension.CumulVar(routeIndex));
+                summaries.Add(new Summary
+                {
+                    VehicleName = vehiclesNames[vehicleIndex],
+                    Load = (int)load,
+                    Distance = (int)routeDistance,
+                    Time = (int)vehicleSpentTime,
+                    NumberOfVisits = numberOfVisits
+                });
+            }
+
+            return summaries;
+        }
+
+        private IList<Totals> GetTotals(FileInput input, RoutingIndexManager manager, RoutingModel routing, Assignment solution)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unused method parameter 'manager'.](https://app.codacy.com/app/iamprovidence/CVRPWT/pullRequest?prid=4635982)